proto/attestation/reference_value.proto

```diff
@@ -46,57 +47,35 @@ message VerifyAmdSevAttestationReport {
   // If true, will skip the check that the TEE is not in debug mode.
   bool allow_debug = 3;
 
-  optional VerifyLogEntry stage0_binary = 4;
+  optional LogEntryVerification stage0_log_entry = 4;
 }
 
-message VerifyIntelTdxAttestationReport {
+message IntelTdxReferenceValues {
   // TBD
 }
 
 // Verifies the log entry, including signatures and the digest.
-message VerifyLogEntry {
+message LogEntryVerification {
   // The endorser’s public signing key for signature verification.
   bytes endorser_public_key = 1;
 
   // Rekor’s public signing key for signature verification.
   bytes rekor_public_key = 2;
 }
 
-// Verifies that the field contains at least one of the given digests.
-// No checks are performed if this is empty.
-message Measurements {
-  repeated RawDigest digests = 1;
+// As measured by stage0
+message KernelLayerReferenceValues {
+  optional LogEntryVerification log_entry = 1;
+  optional RawDigest kernel = 2;
```

Comment:
Still not sure about these two, the naming does not really help here. LogEntry is the type of verification, but the field name should convey what it is verified.

Reply:
The field names are derived from the CBOR label name, by omitting `Measurements` from the end. Kernel and generic binary have different CBOR label names, so they have also different field names. Not optimal - fixed by comment for now.